lib/ably/models/token_details.rb

```diff
@@ -95,10 +95,15 @@ def client_id
     # Returns true if token is expired or about to expire
     # For tokens that have not got an explicit expires attribute expired? will always return true
     #
+    # @param attributes [Hash]
+    # @option attributes [Time] :from   Sets a current time from which token expires
+    #
     # @return [Boolean]
-    def expired?
+    def expired?(attributes = {})
```

Comment:
Why not use the default option?
```
def expired?(from: Time.now)
```

Reply:
Such syntax is available since ruby `2.x` but we support `1.9.3+` https://github.com/ably/ably-ruby#supported-platforms

```markdown
This SDK supports Ruby 1.9.3+. For eventmachine and Ruby 3.0 note...
```

I don't know if anyone use this lib on the machine with ruby `1.9.x`